example-locale_api/README.md

```diff
@@ -0,0 +1,155 @@
+
+```js
+import * as Appwrite from "appwrite";
+
+let sdk = new Appwrite();
+
+sdk
+  .setEndpoint("[API ENDPOINT]") // Your API Endpoint
+  .setProject("[Your Project ID]"); // Your project ID
+
+export {sdk};
+```
+
+A deeper inspection of this code can be found in the comments within it,
+
+TL:DR: Create a appwrite SDK Instance and initalise it with the endpoint and ProjectID of the project we are working with then export this for usage outside of the file.
+
+## Creating App.vue
+
+Inside our `src/App.vue` we'll be importing from our `utils.js`.
+Paste the following code inside our `src/App.vue` file.
+
+```js
+<template id="app">
+  <div>
+    <h1 class="header">app<strong>write</strong> Locale API example</h1>
+    <div class="info">
+      <h1>Here are you Location Details!</h1>
+      <p>
+        <strong>Your IP Address</strong>
+        : &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;{{ info.ip }}
+      </p>
+      <p>
+        <strong>Your Country</strong>
+        : &nbsp;&nbsp;&nbsp;{{ info.country }}
+      </p>
+      <p>
+        <strong>Your Country Code</strong>
+        : &nbsp;&nbsp;&nbsp;{{ info.countryCode }}
+      </p>
+      <p>
+        <strong>Your Continent</strong>
+        : &nbsp;&nbsp;&nbsp;{{ info.continent }}
+      </p>
+      <p>
+        <strong>Your Currency</strong>
+        : &nbsp;&nbsp;&nbsp;{{ info.currency }}
+      </p>
+    </div>
+  </div>
+</template>
+
+<script>
+import { sdk } from "./utils";
+export default {
+  name: "App",
+  data() {
+    return {
+      info: {},
+    };
+  },
+  method: {
+    fetchApiData() {
+      let promise = sdk.locale.get();
+      promise.then(
+        function () {
+          this.setResults; // Success
+        },
+        function (error) {
+          console.log(error); // Failure
+        }
+      );
+    },
+    setResults(results) {
+      this.info = results;
+    },
+  },
+};
+</script>
+
+<style>
+body {
+  background-color: #171d37;
+}
+#app {
+  font-family: Avenir, Helvetica, Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  text-align: center;
+  color: #2c3e50;
+  margin-top: 60px;
+}
+.header {
+  text-align: center;
+  font-family: "Poppins", sans-serif;
+  color: #f35884;
+  margin-left: 10%;
+}
+.info {
+  text-align: center;
+  margin-left: 35%;
+  width: 500px;
+  height: 400px;
+  background-color: white;
+  border-radius: 10px;
+  box-shadow: 0px 0px 10px #f35884;
+  font-family: "Poppins", sans-serif;
+}
+.info h1 {
+  padding-top: 50px;
+}
+</style>
+```
+
+## Explanation of above code:
+
+We first initialized the SDK in our `utils.js` file and then imported the locale.get() method into our `App.vue` file.
+Inside our methods section, we then by using a promise flooded the `info` data with the API's response.
```

Comment:
That last past of the sentence is somewhat weird in my opinion.

Reply:
@Kolkies , I've changed that part, can you please review it again. Thanks